### DISCUSSION TITLE
個人詳細画面のデザインについて

### DISCUSSION THREAD

**Discussion by yu-3in** [jphacks/F_2208#81] [Ideas]
## 決めること
- [x] どの要素を配置するか
- [x] レイアウトをどうするか
- [x] 背景に画像を用いるか
  - [x] 画像を用いる場合は、どのような画像か

**Comment by sakai1250** [comment]
自分がユーザーだとしたら、個人詳細画面では、完了したタスク履歴と、ポイント履歴を見たいです。
